src/platform/datapath_raw.h

```diff
@@ -253,27 +238,25 @@ CxPlatDpRawTxEnqueue(
 // Raw Socket Interface
 //
 
-typedef struct CXPLAT_SOCKET {
+typedef struct CXPLAT_SOCKET_RAW {
 
     CXPLAT_HASHTABLE_ENTRY Entry;
     CXPLAT_RUNDOWN_REF Rundown;
-    CXPLAT_DATAPATH* Datapath;
+    CXPLAT_DATAPATH_RAW* RawDatapath;
     SOCKET AuxSocket;
```

Comment:
Put a comment that this is only used for QUIC over TCP? (it is, right?)

Reply:
No, This can be any raw socket. XDP, DPDK and QUIC over TCP (XDP)